bay/images/docker/settings.php

```diff
@@ -101,21 +102,42 @@
       $redis->auth($redis_password);
     }
 
-    $response = $redis->ping();
-    if (strpos($response, 'PONG') === 'FALSE') {
+    // Check the redis mode - to determine which interface we use to connect.
+    $info = $redis->info();
```

Comment:
This might be expensive on each origin hit, locally it seemed okay but we should test in cluster before roll out.

Reply:
How long (in ms) does this take? Would be keen to measure that